mobile/src/screens/notification-settings/libs/constants.ts

```diff
@@ -0,0 +1,5 @@
+const NOTIFICATION_SETTINGS_DEFAULT_VALUE = {
+  notification: true,
```

Comment:
boolean variables shout start with "yes" prefix
```suggestion
  hasNotification: true,
```